parm/config/gefs/config.base.emc.dyn

```diff
@@ -154,35 +154,27 @@ export OPS_RES="C768" # Do not change
 export LEVS=128
 export CASE="@CASECTL@"  # CASE is required in GEFS to determine ocean/ice/wave resolutions
 export CASE_ENS="@CASEENS@"
-# TODO: This should not depend on $CASE or $CASE_ENS
-# These are the currently available grid-combinations
+export OCNRES="@OCNRES@"
+export ICERES="${OCNRES}"
+# These are the currently recommended grid-combinations
 case "${CASE}" in
     "C48")
-        export OCNRES=500
         export waveGRD='glo_500'
```

Comment:
If we're decoupling the marine resolution from the atmosphere resolution, should we also decouple the wave grid?

Reply:
some other time.